lighthouse-core/lib/i18n/i18n.js

```diff
@@ -56,9 +56,13 @@ const UIStrings = {
   displayValueMsSavings: 'Potential savings of {wastedMs, number, milliseconds}\xa0ms',
   /** Label for a column in a data table; entries will be the URL of a web resource */
   columnURL: 'URL',
-  /** Label for a column in a data table; entries will be the size of a web resource in kilobytes. */
+  /** Label for a column in a data table; entries will be the size of something resource (generic). */
   columnSize: 'Size',
+  /** Label for a column in a data table; entries will be the transfer size (on the wire) of a web resource in kilobytes. */
+  columnResourceSize: 'Resource Size',
   /** Label for a column in a data table; entries will be the time to live value of the cache header on a web resource. */
+  columnTransferSize: 'Transfer Size',
+  /** Label for a column in a data table; entries will be the resource size (post-decoding) of a web resource in kilobytes. */
```

Comment:
The previous description's "number of kilobytes transferred to load a set of files" (and something like "the size of a file in kilobytes") might be more meaningful to translators than "on the wire" and "post-decoding".

Reply:
switched to "file size" and "download size" in these strings